### ISSUE TITLE
fully support Xen (under RHEL7 and clones)

### ISSUE THREAD

**Issue by ktaraszk** [hypervm-ng/hypervm-ng#33]
HyperVM has to support Xen at RHEL7 and its clones (ie CentOS 7)

**Comment by ktaraszk** [comment]
Isn't it under testing now as of https://github.com/hypervm-ng/hypervm-ng/pull/31

**Comment by dkstiler** [comment]
Yes Its not working at the current state, VM are not able to boot. I need the errors and whatever issues are there to fix them.

**Comment by tgd1973** [comment]
I am happy for you, that you have resumed work on this wonderful, but forgotten software. I can't help much but I assure you that I closely follow the way they develop them.
I hope that with regard to Centos, alternatives for another OS will be found.

**Comment by dkstiler** [comment]
Well with the above commit #34  seems we have a properly working xen system acting as hypervisor, When the bridge model networking is configured properly within xen configurations. 
There are  yet a few bugs we need to iron out. One of them its when you enable the use of swap, on xen LVM setup then it does not able to find it and assign it to the VM.
We also need some testing for the File version of xen storage vs LVM
